WORKSPACE

```diff
@@ -79,3 +79,37 @@ BAZEL_VERSION = "3.4.1"
 BAZEL_INSTALLER_VERSION_linux_SHA = "9808adad931ac652e8ff5022a74507c532250c2091d21d6aebc7064573669cc5"
 
 BAZEL_INSTALLER_VERSION_darwin_SHA = "b168b9c4186916cd07922b1155bca14eecc812729669f1fdbab141f3f4eee2a0"
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "io_bazel_rules_rust",
+    strip_prefix = "rules_rust-fdf9655ba95616e0314b4e0ebab40bb0c5fe005c",
+    urls = [
+        # Master branch as of 2019-10-07
+        "https://github.com/bazelbuild/rules_rust/archive/fdf9655ba95616e0314b4e0ebab40bb0c5fe005c.tar.gz",
+    ],
+)
```

Comment:
Is there a reason why these archives must be in WORKSPACE? Can you move them to `third_party/externals.bzl` where all the rest of the third party dependencies are? (we have third party deps in a separate function to make it easier to depend on sorbet as a bazel library and get the same deps)

Reply:
I was following the guide from io_bazel_rules_rust, will update